README.rst

```diff
@@ -79,6 +79,24 @@ Detecting events::
 
     #set polarity to 1 on start:
     PWM.start("PWM0", 50, 2000, 1)
+    
+**SOFTPWM**::
+
+    import CHIP_IO.SOFTPWM as PWM
+    #PWM.start(channel, duty, freq=2000, polarity=0)
+    #duty values are valid 0 (off) to 100 (on)
+    #you can choose any pin
+    PWM.start("XIO-P7", 50)
+    PWM.set_duty_cycle("PWM0", 25.5)
+    PWM.set_frequency("PWM0", 10)
+
+    PWM.stop("PWM0")
+    PWM.cleanup()
+
+    #set polarity to 1 on start:
+    PWM.start("PWM0", 50, 2000, 1)
+    
```

Comment:
These probably need to change to XIO-P7?  I'm assuming this would work on any of the open GPIO pins on the CHIP.